modules/cluster-logging-collector-tuning.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-collector.adoc
+
+[id="cluster-logging-collector-tuning_{context}"]
+= Advanced configuration for the log collector
+
+Cluster logging includes multiple Fluentd parameters that you can use for performance tuning. The parameters involve the Fluentd chunking behaviors.
+
+Fluentd collects log data into _chunks_. When Chunks are created, they are consider to be in the _stage_, where chunks get filled with data. When the chunk is full, Fluentd moves the chuck to the _queue_, where chunks are held before being flushed, or written out to their destination. Fluentd can fail to flush a chunk for a number of reasons, such as network issues or capacity issues at the destination. If a chunk cannot be flushed, Fluentd retries flushing as configured.
+
+By default in {product-title}, Fluentd uses the _exponential backoff_ retry method, where Fluentd doubles the time it waits between attempts to retry flushing, which helps reduce re-connect requests with the destination. You can disable exponential backoff and use the _periodic_ retry method instead, which retries writing off the chunks at a specified interval. 
+
+By default, Fluentd retries flushing the chunks indefinitely. You can change this behavior and discard chunks that cannot be properly flushed.
```

Comment:
@periklis retry_forever ??
From Fluentd docs:
_Fluentd will abort the attempt to transfer the failing chunks...
In these events, all chunks in the queue are discarded. If you want to avoid this, you can enable retry_forever to make Fluentd retry indefinitely._

https://docs.fluentd.org/buffer#handling-successive-failures

Reply:
I removed this line based on your comment below about `retry_forever`.